webapp/platform/client/package.json

```diff
@@ -35,8 +39,5 @@
     "test": "jest",
     "test-ci": "jest --ci --forceExit --detectOpenHandles --maxWorkers=100%",
     "clean": "rm -rf lib node_modules tsconfig.tsbuildinfo"
-  },
-  "devDependencies": {
-    "jest": "27.1.0"
```

Comment:
awesome we were able to remove this dependency, this was added because of a type issue with jest when we updated last time